backend/scripts/start-ssh-tunnel.sh

```
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# SSH 터널 시작 스크립트
+# .env 파일의 환경 변수를 사용하여 SSH 터널을 생성합니다.
+
+set -e
+
+# .env 파일 로드
+if [ -f .env ]; then
+    export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)